packages/node_modules/@ciscospark/internal-plugin-presence/test/integration/spec/presence.js

```diff
@@ -40,6 +40,29 @@ describe('plugin-presence', function () {
 
     afterEach('deregister spock with mercury', () => spock.spark.internal.mercury.disconnect());
 
+    describe('#isEnabled()', () => {
```

Comment:
I would prefer to see this as its own test and verifying from the features instead of relying on the other  `#enable` and `#disable` functions. As well as having the `#enable` test not rely on `#isEnabled`.  It feels somewhat self verifying instead of unit testing each function. Not really a blocker though, just more of a personal preference.

Reply:
@adamweeks so even though they're really helper methods that call the features methods, you would explicitly use the features methods in the tests?